blockchain_connector/ethereum/setup.py

```diff
@@ -26,16 +26,17 @@
     sys.exit(1)
 
 
-setup(name='avalon_blockchain_connector',
+setup(name='avalon_ethereum_connector',
       version=subprocess.check_output(
-        ['../bin/get_version']).decode('utf-8').strip(),
-      description='Avalon Blockchain Connector',
+        ['../../bin/get_version']).decode('utf-8').strip(),
+      description='Ethereum Connector',
```

Comment:
I would call it `Avalon Ethereum Connector` to make it clear this connects Avalon and Ethereum and is not an Ethereum-only component.

Reply:
Done